components/campaign/ChaosToken.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { ImageBackground, StyleSheet, Text, View } from 'react-native';
+import { ChaosTokenType } from '../../constants';
+import ChaosTokenIcon from '../core/ChaosTokenIcon';
+import { iconSizeScale } from '../../styles/space';
+import DeviceInfo from 'react-native-device-info';
```

Comment:
If you don't mind, I like to keep the npm package imports at the top, with the local ones down below.  A few of the 'ttag' rules don't follow this because I did a global find replace, but I try to fix them when I come across them.